src/Components/Header/Header.scss

```diff
@@ -1,4 +1,18 @@
-.pf-v5-c-page__main-section .pf-v5-c-toolbar__group {
+.widg-c-page__main-section--header,
+.widg-c-split--add-widget {
+  max-width: 1280px;
```

Comment:
We may need to revisit this value as I'm not sure this is high enough or necessary (I can see the config drop down and add widget button have shifted over in your screenshot).

Reply:
@florkbr If it's not high enough we can change it later. And I think it is necessary, unless you know another way to accomplish what I'm trying to do.